src/subscription/PaymentViewer.js

```diff
@@ -96,8 +96,9 @@ export class PaymentViewer implements UpdatableSettingsViewer {
 			label: "paymentMethod_label",
 			click: createNotAvailableForFreeClickHandler(true, () => {
 					if (this._accountingInfo) {
-						let lastPosting = neverNull(this._postings[0])
-						let nextPayment = Number(lastPosting.balance) * -1
+						let nextPayment = this._postings.length
+							? Number(this._postings[0].balance) * -1
```

Comment:
maybe `filterInt()` instead?

Reply:
I am not sure why we would need `filterInt()`? I think it is safe to assume that `balance` is a NumberString.

Reply:
Probably, yeah. Leave it then.